src/seedu/addressbook/AddressBook.java

```diff
@@ -1136,26 +1127,24 @@ private static String getUsageInfoForExitCommand() {
                 + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_EXIT_EXAMPLE);
     }
 
-
     /*
-     * ============================
-     *         UTILITY METHODS
-     * ============================
+     * ============================ UTILITY METHODS ============================
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
```

Comment:
the comment here still refers to addressbook, no? (think where else in general would you use `p/ , d/` ?
when you update the behavior of the method, please ensure you update the document to reflect the behavior.

in this particular case, intention was to make remove prefix method as generic as possible, definitely that should be reflected in the header comments